magmartfront/src/app/admin/pokemon/create/page.tsx

```diff
@@ -29,7 +29,7 @@ export default function CadPokemon() {
 
     try {
       // Envia os dados do Pokémon para a API
-      const response = await fetch(`${process.env.API_URL}/pokemon/new`, {
+      const response = await fetch('http://localhost:3001/pokemon/new', {
```

Comment:
Manter uso do `process.env`
```suggestion
      const response = await fetch('${process.env.API_URL}/pokemon/new', {
```